ginga/cmap.py

```diff
@@ -11476,23 +11476,15 @@ def add_matplotlib_cmap(cm, name=None):
 
 def add_matplotlib_cmaps():
     """Add all matplotlib colormaps."""
-    import matplotlib.pyplot as plt
+    from matplotlib import cm as _cm
```

Comment:
Note: There is actually a *real* `matplotlib._cm` module that is not the same as `matplotlib.cm` but I renamed the import here as not to clash with your existing `cm` *variable* name. I don't think this is an issue because we will never need the actual `matplotlib._cm` module anyway.